math/src/field/fields/u64_prime_field.rs

```diff
@@ -1,13 +1,13 @@
-use crate::cyclic_group::IsCyclicBilinearGroup;
+use crate::cyclic_group::{HasPairing, IsCyclicGroup};
 use crate::field::element::FieldElement;
-use crate::field::traits::HasFieldOperations;
+use crate::field::traits::IsField;
 
 /// Type representing prime fields over unsigned 64-bit integers.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
 pub struct U64PrimeField<const MODULO: u64>;
 pub type U64FieldElement<const ORDER: u64> = FieldElement<U64PrimeField<ORDER>>;
```

Comment:
Is there a reason why we can't do this so we can have only one word for the same concept?
```suggestion
pub struct U64PrimeField<const ORDER: u64>;
pub type U64FieldElement<const ORDER: u64> = FieldElement<U64PrimeField<ORDER>>;
```

Reply:
No reason. Changed it.